index.html

```diff
@@ -469,39 +469,71 @@ <h2>Validated address information</h2>
       address.events.on("post-enrichment", function(data) {
         if ((data.result.geocodes && data.result.geocodes.latitude) ||
             (data.result.what3words && data.result.what3words.latitude)) {
-          document.querySelector("#map").classList.remove("hidden");
-          
-          let lat, long;
-          if (data.result.what3words.latitude && data.result.what3words.longitude && data.result.what3words.name) {
+          document.querySelector("#map").classList.remove("hidden");          
+          document.querySelector(".metadata #what3words-key").classList.add("hidden");
+          document.querySelector(".metadata #what3words-value").classList.add("hidden");
+
+          let w3wLat, w3wLong, w3wLatLong;
+          if (data.result.what3words && data.result.what3words.latitude) {            
             document.querySelector(".metadata #what3words-key").classList.remove("hidden");
             document.querySelector(".metadata #what3words-value").classList.remove("hidden");
             document.querySelector(".metadata #what3words-value").innerHTML = '///' + data.result.what3words.name;
             
-            lat = data.result.what3words.latitude;
-            long = data.result.what3words.longitude;
-          } else {
-            document.querySelector(".metadata #what3words-key").classList.add("hidden");
-            document.querySelector(".metadata #what3words-value").classList.add("hidden");
-            
-            lat = data.result.geocodes.latitude;
-            long = data.result.geocodes.longitude;
-          }
+            w3wLat = data.result.what3words.latitude;
+            w3wLong = data.result.what3words.longitude;
+            w3wLatLong = [w3wLat, w3wLong];
+          } 
+          
+          let geoLat, geoLong, geoLatLong;
+          if (data.result.geocodes && data.result.geocodes.latitude) {
+            geoLat = data.result.geocodes.latitude;
+            geoLong = data.result.geocodes.longitude;
+            geoLatLong = [geoLat, geoLong];
+          } 
 
-          var latlong = [lat, long];
           var zoom = 16;
           var attribution = '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors';
 
+          // Define a custom icon for what3words
+          var w3wIcon = L.icon({
+            iconUrl: './dist/images/w3w.loc.png',
+            iconSize:     [30, 37], // size of the icon
+            iconAnchor:   [15, 36], // point of the icon which will correspond to marker's location
+            popupAnchor:  [0, -20] // point from which the popup should open relative to the iconAnchor
+          });
 
-          // Intantiate a new map
+          // Instantiate a new map
           if (!addressValidationMap) {
-            addressValidationMap = L.map('map').setView(latlong, zoom);
+            addressValidationMap = L.map('map').setView([51.500264, 0.633506],zoom);

```

Comment:
In the middle of Thames? :)

Reply:
Why has this lat/long been hardcoded?

Reply:
I tired to just set the zoom, but that threw an error, so I went with something :)

Reply:
I assume it gets "overwritten" when/if the map pans to a w3w or geo marker.
Maybe a quick comment would've solved the confusion as I genuinely thought you'd mistakenly committed some test code.